.github/workflows/run_tests.yml

```diff
@@ -19,15 +19,27 @@ on: [pull_request, push]
 env:
   # See https://dev.to/dtinth/caching-docker-builds-in-github-actions-which-approach-is-the-fastest-a-research-18ei.
   CACHE_TARGET: "docker.pkg.github.com/open-reaction-database/ord-editor/editor-cache"
+  ORD_SCHEMA_TAG: v0.2.6
 
 jobs:
   test_editor:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
-    - uses: actions/setup-python@v2
+    - uses: actions/checkout@v2
```

Comment:
Redundant line?

Reply:
That line is checking out ord-schema, not ord-editor; so not redundant.